packages/atlas/src/views/viewer/ChannelView/ChannelView.styles.ts

```diff
@@ -131,20 +134,17 @@ export const PaginationContainer = styled.div`
 
 export const TabsContainer = styled.div`
   display: grid;
+  padding-top: ${sizes(8)};
   margin-bottom: ${sizes(8)};
   gap: ${sizes(2)};
   grid-template: 'tabs tabs tabs' 1fr 'search search search' auto 'sort sort sort' auto / 1fr 1fr;
   align-items: baseline;
 
-  ${media.xs} {
-    padding-top: ${sizes(8)};
-  }
-
   ${media.sm} {
     align-items: center;
-    border-bottom: solid 1px ${oldColors.gray[700]};
+    box-shadow: ${cVar('effectDividersBottom')};
     gap: ${sizes(8)};
-    grid-template: 1fr / auto 1fr 250px;
+    grid-template: 1fr / auto 1fr 193.333px;
```

Comment:
Why `193.333`? Where does it come from?

Reply:
250px was the size of the sort select + label but they removed the label so 193.333 is the remaining size the select should have without it

Reply:
This looks odd, and I still don't see the purpose of setting this to exactly 193.333 px.  
Why can't we just set this to a rounded value like 200px?

Reply:
So, made it 160px as per design, as it no longer has a value label